app/src/main/java/org/mozilla/focus/FocusApplication.kt

```diff
@@ -18,6 +18,8 @@ class FocusApplication : LocaleAwareApplication() {
     lateinit var visibilityLifeCycleCallback: VisibilityLifeCycleCallback
         private set
 
+    val components by lazy { Components() }
```

Comment:
> Is it because we expect it to take the ApplicationContext as an argument in the future? That seems fine then.

Yes, that's exactly the reason. It's going to be `Components(applicationContext)` because some components will need the application context (#959, #960).